awswrangler/redshift.py

```diff
@@ -414,6 +416,7 @@ def _create_table(  # pylint: disable=too-many-locals,too-many-arguments,too-man
     )
     _logger.debug("Executing create table query:\n%s", sql)
     cursor.execute(sql)
+    _logger.debug("Created table %s", table)
```

Comment:
Do we want to make this an INFO log rather than debug? We discussed logging resource creations and deletions in AWS as info rather than debug.

Reply:
Thanks, adgreed